deploy/helm/templates/rbac.yaml

```diff
@@ -0,0 +1,82 @@
+{{- if .Values.serviceAccount.create -}}
+---
```

Comment:
I decided to follow the Helm chart best practices of allowing users to specify a manually created service account.